src/simularium/RemoteSimulator.ts

```diff
@@ -255,53 +260,52 @@ export class RemoteSimulator implements ISimulator {
         return `wss://${this.serverIp}:${this.serverPort}/`;
     }
 
-    private connectToUriAsync(address): Promise<string> {
-        const connectPromise = new Promise<string>((resolve) => {
-            this.connectToUri(address);
-            resolve("Successfully connected to uri!");
-        });
-
-        return connectPromise;
-    }
```

Comment:
After discussing WebSocket object creation with @EricJIsaac , I thought this function was unnecessary, since it's an async wrapper for a function (`connectToUri`) which doesn't have any async operations?